_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: otel_logs_source
```

Comment:
```suggestion
title: OTel logs source 
```

Reply:
@dlvenable Keeping underscores in the title will impact SEO and accessibility. Also, #3802 was created to address titles for these plugins as well.